db/migrations/20190712145900_create_buckets.cr

```diff
@@ -0,0 +1,17 @@
+class CreateBuckets::V20190712145900 < Avram::Migrator::Migration::V1
```

Comment:
Can you give this a descriptive name like `CreateBucketsWithArrayTypes` so it's clear from the filename it is testing array types?